moqt-server/src/modules/pubsub_relation_manager/commands.rs

```diff
@@ -41,11 +42,16 @@ pub(crate) enum PubSubRelationCommand {
         track_name: String,
         resp: oneshot::Sender<Result<bool>>,
     },
-    GetUpstreamSubscription {
+    GetUpstreamSubscriptionByFullTrackName {
         track_namespace: Vec<String>,
         track_name: String,
         resp: oneshot::Sender<Result<Option<Subscription>>>,
     },
+    GetDownstreamSubscriptionByIds {
```

Comment:
This function is searching with session_id and subscribe_id.
So that please make function name same as `GetRequestingDownstreamSessionIdsAndSubscribeIds`.


And, why is this function named with`Ids`? I think it is good to use `Id`, not `Ids`.
(Please change also L59 `GetRequestingDownstreamSessionIdsAndSubscribeIds`)

Reply:
I will change it to `GetRequestingDownstreamSessionIdAndSubscribeId`.